src/index.js

```diff
@@ -130,11 +130,14 @@ function raw (service, method, params) {
   // handle client methods like indices.create
   const [meth, ext] = method.split('.');
 
-  if (typeof ext !== 'undefined') {
-    return service.Model[meth][ext](params);
+  if (typeof service.Model[meth] === 'undefined') {
+    return Promise
+      .reject(new Error(`There is no query method ${meth}`));
```

Comment:
We should be using Feathers errors rather than generic ones. `errors.GeneralError` would be suitable here, I think. Similarly in line 121.